### PR TITLE
Tidy up & remove duplicate + dead links

### PR DESCRIPTION
**Pull request by mountainash**
Just some general house-cleaning...

My "goal" was to remove the 3 broken Polygon links from the Showcase page at https://chakra-ui.com/community/showcase (the *Submit your project* link point to this repo) - but it doesn't appear like the Showcase is automatically generated from the content of this project. Maybe whichever maintainer to read this also has access to the Showcase source?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,17 +138,16 @@ A collection of Chakra UI-related awesomeness
 - [How to Set Up Chakra UI with Next JS by creating a Hero component](https://creativedesignsguru.com/chakra-ui-next-js/)
 - [Choosing the right component library for your design system: MUI vs Chakra](https://engineering.udacity.com/choosing-the-right-component-library-for-your-design-system-mui-vs-chakra-45c4c949d150)
 - [How To Build an Advanced Multipart Component with Chakra UI](https://www.lekoarts.de/react/how-to-build-an-advanced-multipart-component-with-chakra-ui)
-- [Choosing the right component library for your design system: MUI vs Chakra](https://engineering.udacity.com/choosing-the-right-component-library-for-your-design-system-mui-vs-chakra-45c4c949d150)
```
Comment (nikolovlazar) [inline README.md:144]: This is not a dead link. The article exists.

File: contributing.md [hunk 2]
```diff
@@ -4,16 +4,6 @@ Please note that this project is released with a
...
----
-
-Ensure your pull request adheres to the following guidelines:
-
-- Make sure you take care of this
-- And this as well
-- And don't forget to check this
-
-Thank you for your suggestions!
-
```
Comment (nikolovlazar) [inline contributing.md:16]: Good catch!

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Thanks for working on this @mountainash 🙌! I left you one comment about a deleted article that still exists, and I can also see there are some merge conflicts as well. Could you take care of that?

File: README.md (hunk 1 above)
Comment (mountainash) [reply README.md:144]: Yes... but it's a duplicate of the same link two list-items above.

**Comment by mountainash** [comment]
Hello @nikolovlazar I've made the requested changes and updated the branch from the upstream HEAD ready for merge, then updated all the new changes as well.

Do you know if this will fix the original intention of cleaning up the broken links on https://chakra-ui.com/community/showcase ?

File: README.md (hunk 1 above)
Comment (nikolovlazar) [reply README.md:144]: Ah, got it!

**Review by nikolovlazar — APPROVED** [review]
LGTM!

**Comment by nikolovlazar** [comment]
@mountainash yes it will!

**Comment by mountainash** [comment]
> yes it will!

Doesn't seem to have changed... might want to check out if any automatons are broken?

**Comment by nikolovlazar** [comment]
@mountainash the `update sponsors, members, showcase and avatars` PR will pick it up tomorrow morning: https://github.com/chakra-ui/chakra-ui-docs/pull/1006

**Comment by nikolovlazar** [comment]
Don't worry. That's the one we need. Here's the action if you want to see everything that's happening: https://github.com/chakra-ui/chakra-ui-docs/actions/runs/3109570238/jobs/5039893254

**Pull request merged**